app/assets/stylesheets/elements/_flash-messages.scss

```diff
@@ -0,0 +1,22 @@
+.flash {
+  background-color:$highlight-colour;
+  border-width:0 0 0 8px;
+  border-style:solid;
+  margin: $gutter 0;
+  padding:$gutter 0;
+  &.info {
+    border-color:$grass-green;
```

Comment:
*FLASH MESSAGE COPY *


*IF EMAIL ADDRESS SUPPLIED (coming from any page, having clicked 'save and
complete later')*

Claim saved. Your claim number has been emailed to you. *Return to your
claim* [Link]


*IF NO EMAIL ADDRESS SUPPLIED (coming from interstitial page)*

You have been logged out. *Return to your claim.* [Link]


*NOTE, NEW COPY / DESIGN FOR INTERSTITIAL PAGE (IF NO EMAIL ADDRESS
SUPPLIED) IN* GOOGLE DOC
<https://docs.google.com/a/digital.justice.gov.uk/document/d/1h0V_N7wWygMceT7bmTptDoK1WTW0DeMmWqNOIKX_i08/edit>



Michael Elwin
Digital Content Designer
07947 557671

On 27 January 2015 at 22:38, Stevie Graham <notifications@github.com> wrote:

> In app/assets/stylesheets/elements/_flash-messages.scss
> <https://github.com/ministryofjustice/atet/pull/589#discussion_r23649895>:
>
> > @@ -0,0 +1,22 @@
> > +.flash {
> > +  background-color:$highlight-colour;
> > +  border-width:0 0 0 8px;
> > +  border-style:solid;
> > +  margin: $gutter 0;
> > +  padding:$gutter 0;
> > +  &.info {
> > +    border-color:$grass-green;
>
> Green isn't working for me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ministryofjustice/atet/pull/589/files#r23649895>.
>